docs/configuration/lineage.rst

```diff
@@ -9,7 +9,7 @@ and virtualenv execution methods (read `execution modes <../getting_started/exec
 
 To emit lineage events, Cosmos can use one of the following:
 
-1. Airflow 2.7 `built-in support to OpenLineage <https://airflow.apache.org/docs/apache-airflow-providers-openlineage/1.0.2/guides/user.html>`_, or
+1. Airflow official OpenLineage provider <https://airflow.apache.org/docs/apache-airflow-providers-openlineage/1.0.2/guides/user.html>`_, or
```

Comment:
I think there is a backtick missing somewhere here for the external link since it looks like this:

 
![image](https://github.com/astronomer/astronomer-cosmos/assets/79104794/7c5d1e89-8f28-43f5-86e6-2e8fb59c520c)

Reply:
Thanks, @jbandoro , great catch - just fixed!